src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -258,7 +258,8 @@ private static boolean isTinymceStyleAttribute(Node attribute) {
     }
 
     private static boolean isHandsontableAttribute(Node attribute) {
-        return checkForAttributeWithSpecificValue(attribute, "id", "spreadsheet");
+        return checkForAttributeWithSpecificValue(attribute, "id", "enrollSpreadsheet")
+                || checkForAttributeWithSpecificValue(attribute, "id", "dataSpreadsheet");
```

Comment:
Can we give them the same class name and check for that instead?

Reply:
If would potentially clash with https://github.com/TEAMMATES/teammates/pull/8884#discussion_r195302082 if there is still a  distinction between the two spreadsheets.